src/AppInstallerCLICore/Workflows/ConfigurationFlow.cpp

```diff
@@ -203,12 +203,12 @@ namespace AppInstaller::CLI::Workflow
                         auto previousThreadGlobals = installDscContext.SetForCurrentThread();
 
                         Manifest::ManifestInstaller dscInstaller;
-// TEMP: Until DSCv3 support is not experimental, allow the preview build to be installed automatically.
-// #ifndef AICLI_DISABLE_TEST_HOOKS
+
+#ifndef AICLI_DISABLE_TEST_HOOKS

```

Comment:
Using a different version of DSC for the tests doesn't sound right to me. Why not just use the stable one all the time?

Reply:
Did you mean `WINGET_ENABLE_RELEASE_BUILD`?

Reply:
I guess we just did not want to maintain a new flag at that time (AICLI_DISABLE_TEST_HOOKS has been overloaded for multiple purposes over time, it essentially becomes non-release vs release).

Since we are actively developing with dscv3 (dscv3 is in development too), we needed preview package for development, we'll make sure all needed features are in dscv3 release before our release.